src/projects.md

```diff
@@ -1,23 +1,99 @@
 ---
 title: Our Projects
-date: 2015-11-03T02:55:22-07:00
-author: elina
+date: 2020-10-11T01:15:35+00:00
+author: Jess Sand
 layout: page
 ---
 
-In response to member feedback, OpenOakland is reevaluating how our projects are vetted and adopted, with a focus on ensuring all projects are aligned with [OpenOakland’s values](https://openoakland.org/our-values) and are truly serving the communities they impact. The projects listed here are currently active but will be going through this new process as it’s developed. If you’d like to contribute to or provide feedback on any of these projects or the evaluation process itself, consider attending our [next Hack Night](https://www.meetup.com/OpenOakland/events/), joining the conversation in our [Slack workspace](http://slack.openoakland.org/), or emailing our [Steering Committee](mailto:steering@openoakland.org).
+ - [Active projects](#openoakland-projects)
+ - [Decommissioned projects](#decommissioned-projects)
+ - [Becoming an OpenOakland project](#becoming-a-project)
+ - [Providing feedback](#feedback)
 
 ---
+---
 
 <div id="projects">
-  <h2>Active Projects</h2>
+
+  ## <a id="openoakland-projects"></a>Ongoing projects that support OpenOakland's operation
```

Comment:
I think I see what you're doing here with an empty <a> tag containing an "id" attribute, which creates a linkable heading. This strikes me as cumbersome to write and confusing to read. Are there other options to do same with plain markdown in Jekyll? If not, I think it should be pure html.

Reply:
I couldn't find any jekyll-specific examples but will look again or swap out.

Reply:
I'm wondering if markdown headings in Jekyll get id tags by default, based on this https://jekyllrb.com/docs/configuration/markdown/#kramdown

Reply:
Yes, you're right and I've got those working now, thanks!